### ISSUE TITLE
Shift - Arrow is making regions

### ISSUE THREAD

**Issue by 717g301github** [akermu/emacs-libvterm#142]
Why is shift arrow invoking region mark?

**Comment by akermu** [comment]
This doesn't also happen for me with `emacs -Q`. Could you please give me an example config?

**Comment by 717g301github** [comment]
emacs -Q means there is no config.

It does not happen always, it depends on what is moving on screen. I cannot figure out why.

If I switch to copy mode with C-c C-t then shift arrow is marking the text. I think it should not, as it is not Emacs standard behavior.

And I think that maybe copy mode or something similar, gets invoked automatially under some conditions, and then sudddenly shift arrow starts marking regions. 

Example is if I am in full screen with mutt or other application and press shift down or shift up, it invokes marking the region. See screenshot.

![Screenshot](https://gnu.support/files/tmp/2019-08-21/2019-21-08-21:43:13.png "Screenshot")

**Comment by Sbozzolo** [comment]
@gnusupport, have you recently experienced this problem?

**Comment by gnusupport** [comment]
Yes, just the same. I can use shift and arrow up and it starts marking the text. I am using vterm since before few days, latest one.

Other major problem is when using mutt or full screen applications, is that if I mark text with the mouse, the region marked will increase just by moving the mouse down. This is most annoying, as I wish to mark the region and copy it, but mistake is happening and I have to repeat, and be careful, as I never know when it is automatically to mark everything down to the bottom of the screen.

**Comment by Sbozzolo** [comment]
Thanks @gnusupport for your feedback. 

> Yes, just the same. I can use shift and arrow up and it starts marking the text. I am using vterm since before few days, latest one.

I cannot reproduce your problem. Can you provide specific steps to trigger the bug starting from `vterm` in a clean environment (`emacs -Q`)?

**Comment by gnusupport** [comment]
Do you mean the problem I last described? That one can be seen on this video here:
https://gnu.support/files/emacs/vterm/2020-02-21-17:47:32.ogv

I mark something in mutt, and simply don't do anything, everything is marked down to the bottom.

**Comment by gnusupport** [comment]
For the original problem with shift arrow, it is still there. In my opinion it will appear in full text terminal applications.

https://gnu.support/files/emacs/vterm/2020-02-21-17:50:54.ogv

Is it meant to be so?

**Comment by Sbozzolo** [comment]
Thanks for your video. Mouse support is not very satisfying at the moment, so I am not too surprised that there is something that is not working well. I don't think I can easily fix that at the moment. But for the first problem, that should be easy to fix if you can provide clear steps to reproduce it.

**Comment by Sbozzolo** [comment]
What is the output of `C-h k S-<up>` when you are in `VTermCopy` mode?

**Comment by gnusupport** [comment]
I don't think it is related to mouse. It will take place if I don't move mouse.

> <up> (translated from <S-up>) runs the command vterm-send-up (found in
> vterm-mode-map), which is an interactive Lisp closure in ‘vterm.el’.
> 
> It is bound to <up>.
> 
> (vterm-send-up)
> 
> Sends ‘<up>’ to the libvterm.

**Comment by gnusupport** [comment]
* Gabriele Bozzola <notifications@github.com> [2020-02-21 17:23]:
> Thanks @gnusupport for your feedback. 
> 
> > Yes, just the same. I can use shift and arrow up and it starts marking the text. I am using vterm since before few days, latest one.
> 
> I cannot reproduce your problem. Can you provide specific steps to
> trigger the bug starting from `vterm` in a clean environment (`emacs
> -Q`)?

I have tried it, and it is there.

Basically when using full screen terminal application this problem is
there.

I am using mutt for example, that is where I am trying to copy the
text usually, so if I mark the text with mouse, then if I let it be
for 1-3 seconds, many lines get marked automatically later, marking
normally goes down to the bottom of the screen.

I just have the feeling that it could be connected to some refreshing
of the full screen applications.

Jean
